client/src/components/clan/ClanSearchEntry.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { LinkContainer } from 'react-router-bootstrap';
+import { Media, Well, Image } from 'react-bootstrap';
+import { Button, ButtonToolbar, ButtonGroup, Glyphicon } from 'react-bootstrap';
+
+
+class ClanSearchEntry extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  joinClan() {
+    console.log('Friend Added!')
+    //TODO: Add User as Friend
```

Comment:
Friends isnt supposed to be there. These are incomplete components that we needed to push up to appease Travis.